src/lib/storage/zero_suppression/fixed_size_byte_aligned/fixed_size_byte_aligned_encoder.cpp

```diff
@@ -0,0 +1,44 @@
+#include "fixed_size_byte_aligned_encoder.hpp"
+
+namespace opossum {
+
+std::unique_ptr<BaseZeroSuppressionVector> FixedSizeByteAlignedEncoder::encode(
+    const PolymorphicAllocator<size_t>& alloc, const pmr_vector<uint32_t>& vector, const ZsVectorMetaInfo& meta_info) {
+  const auto max_value = meta_info.max_value ? *meta_info.max_value : _get_max_value(vector);
+  return _encode_using_max_value(alloc, vector, max_value);
+}
+
+std::unique_ptr<BaseZeroSuppressionEncoder> FixedSizeByteAlignedEncoder::create_new() const {
+  return std::make_unique<FixedSizeByteAlignedEncoder>();
+}
+
+uint32_t FixedSizeByteAlignedEncoder::_get_max_value(const pmr_vector<uint32_t>& vector) const {
```

Comment:
If the information is not readily available, it's not a "get" but maybe a "find".

Reply:
My naming convention so far has been to name getters that just return a variable without prefix (`int my_variable() const`) and add the prefix `get` to getters that do a bit more (`int get_my_variable() const`). `find` sounds like it needs to search for it in a list of elements. In specific case, that actually applies so I’ll change it.